pkg/client/factory.go

```diff
@@ -60,12 +72,22 @@ func NewFactory(baseName string) Factory {
 		baseName: baseName,
 	}
 
+	// If no namespace is passed in, first try to set it via env var.
+	if f.namespace == "" {
```

Comment:
`f.namespace` will always be empty here since it's not a constructor arg - so let's just remove the `if` statement